metecho/api/migrations/0103_alter_task_status.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.2 on 2021-05-13 15:34
+
+from django.db import migrations, models
+
+from metecho.api.models import TASK_STATUSES
+
+
+def set_task_cancelled_statuses(apps, schema_editor):
+    Task = apps.get_model("api", "Task")
+    for instance in Task.objects.filter(
+        status=TASK_STATUSES["In progress"], pr_is_open=False, pr_number__isnull=False
+    ):
+        instance.status = TASK_STATUSES.Cancelled
+        instance.save()
```

Comment:
@jerivas @davisagli I realize this isn't entirely perfect, because if there are existing Tasks where the PR was closed on GitHub _but then_ another commit was made via Metecho, the status should really be back "In Progress" instead of "Cancelled".

I sort of split the difference here by marking these as "Cancelled" but _not_ setting `review_valid: False` -- which we would do otherwise when the GH PR is closed in real time. I'm open to other solutions here, e.g. not messing with existing tasks at all.

Reply:
@davisagli Do you have an opinion about how aggressive to be in this migration, given that we can't easily replicate the new logic exactly?

Reply:
@jgerigmeyer I think it's fine as is